torch/_inductor/sizevars.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, shape_env=None):
         self.shape_env = shape_env
         self.var_to_val = self.shape_env.var_to_val
         self.guards = []
-        self.replacements: Dict[sympy.Symbol, Expr] = {}
+        self.replacements: Dict[sympy.Symbol, Expr] = self.shape_env.replacements
```

Comment:
do you want an alias, or a copy?

Reply:
Alias

Reply:
might be worth a comment, as the Dict type annot is mutable